pom.xml

```diff
@@ -166,8 +167,7 @@
                     <file>target/classes/plugin.yml</file>
                     <replacements>
                         <replacement>
-                            <token>unknown-version</token>
-                            <!--suppress UnresolvedMavenProperty -->
+                            <token>maven-version-number</token>
```

Comment:
Weird plugin actually, don't understand it 🍵

Reply:
I don't think it's being used. What's the jar name?

Reply:
The comment above says: 
><!--Replace the build number in the plugin.yml -->
Well, it doesn't. 🤡

Reply:
OK, found how it works.
It just really replaces the key to the value.

https://github.com/Multiverse/Multiverse-Inventories/blob/main/src/main/resources/plugin.yml#L3

Can be removed.